tmt/steps/execute/internal.py

```diff
@@ -70,13 +73,11 @@ def options(cls, how=None):
 
     def show(self, keys=None):
         """ Show execute details """
-        keys = (keys or []) + ['script', 'interactive']
-        super().show(keys)
+        super().show(keys or self._keys)
```

Comment:
Hm, I intended this a bit differently when working on `exitfirst`. My plan was to always show/wake the keys that are defined in the class but have an option to show more by adding extra options/keys. This is why the two lists were added together.

Also the exitfirst feature seems to be broken by the change, could you please take a look?

Reply:
The `exit-first` problem should now be fixed.